grid-client/integration_tests/k8s_test.go

```diff
@@ -76,39 +79,54 @@ func TestK8sDeployment(t *testing.T) {
 		err = tfPluginClient.NetworkDeployer.Cancel(context.Background(), &network)
 		require.NoError(t, err)
 	})
+	masterSeed, err := workloads.RandomMyceliumIPSeed()
+	if err != nil {
+		t.Skipf("could not create master mycelium IP seed: %v", err)
+	}
+	workerNodeSeed1, err := workloads.RandomMyceliumIPSeed()
+	if err != nil {
+		t.Skipf("could not create worker Node1 mycelium IP seed: %v", err)
+	}
+	workerNodeSeed2, err := workloads.RandomMyceliumIPSeed()
+	if err != nil {
+		t.Skipf("could not create worker Node2 mycelium IP seed: %v", err)
+	}
 
 	master := workloads.K8sNode{
 		VM: &workloads.VM{
-			Name:        fmt.Sprintf("master_%s", generateRandString(5)),
-			NodeID:      masterNodeID,
-			NetworkName: network.Name,
-			CPU:         minCPU,
-			MemoryMB:    minMemory * 1024,
-			Planetary:   true,
+			Name:           fmt.Sprintf("master_%s", generateRandString(5)),
```

Comment:
you can use the basicVm function for kubernetes, no?

Reply:
basicVm needs public key and it's not used here, also it adds flist which is not necessary in this test

Reply:
you can use empty values, also flists in all nodes will be skipped anyways as long as you add flist to the cluster